lib/preferences.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:shared_preferences/shared_preferences.dart';
+import 'dart:convert';
+
+
+Map<String, bool> _hirasol = {
```

Comment:
This and the below `_katasol` could be named something like `_defaultHiraganaSet`, imo that's more descriptive than the current name

Reply:
Changed!